### DISCUSSION TITLE
Insights script added to head even though disabled on Vercel dashboard

### DISCUSSION THREAD

**Discussion by andrelandgraf** [vercel/community#4566] [Bug Report]
### Page to Investigate

andrelandgraf.dev

### Steps to Reproduce

I am not using Insights (or at least I tried to disable it), but `/_vercel/insights/script.js` is still added to the `head` of my document, and the request fails with 404, resulting in a console error.

```html
<script src="/_vercel/insights/script.js" defer="" data-sdkn="@vercel/analytics" data-sdkv="1.0.2"></script>
```

I may have enabled Insights or at least interacted with the page's content but disabled it after. I now see 404 GET request failures in production, with the Vercel insights script being requested and failing.

The browser console also prints the following message: `[Vercel Web Analytics] Failed to load script from /_vercel/insights/script.js. Be sure to enable Web Analytics for your project and deploy again. See https://vercel.com/docs/concepts/analytics/quickstart for more information.` but the link leads to a 404 page for me.

![Screenshot 2023-10-21 at 11 53 38 AM](https://github.com/vercel/community/assets/15163028/b4240eef-815e-40db-8bf2-ed0a011c70c7)

**Comment by amyegan** [comment]
After disabling Insights, did you redeploy your site?

**Reply by andrelandgraf** [reply]
Yes. Triggered several re-deploys both through the Vercel dashboard and by pushing changes to the repository.

**Comment by andrelandgraf** [comment]
Turns out that I had the npm package installed and rendered the component in my React app. My fault, I was using a template and did not understand that Vercel analytics works via a React component. I was under the impression the script got added through a head injection. Closing!

**Comment by alexiuscrow** [comment]
I had a similar issue. @andrelandgraf, Could you please shed light on what exactly went wrong and how you fix the error?
![Знімок екрана 2024-01-11 о 18 05 04](https://github.com/vercel/community/assets/8084504/4dae2a88-6765-46d2-b0bb-8c23b72fa243)

**Reply by alexiuscrow** [reply]
I found out that for some reason the request is blocked in the Arc browser, but works in Chrome. I feel a bit confused.

**Reply by kapitche** [reply]
It's probably caused by a plugin or extension that blocks incoming traffics from different domains

**Reply by AStox** [reply]
Heh, This was my issue too. Developing in arc and couldn't figure out why the script was failing to load. Thanks!

**Reply by enzo-inc** [reply]
Just had the exact same problem on Arc too, works on Chrome

**Comment by encryptedDegen** [comment]
For anyone that still might be wondering about this, try switching your plugins off one by one and see if the issue gets resolved. For me it was the add blocker I was using in my Chrome browser. 
Hope this helps.
